test_data/assertoor/network_params_blob.yaml

```diff
@@ -0,0 +1,21 @@
+participants:
+  - el_type: ethrex
+    cl_type: lighthouse
+    validator_count: 32
+  - el_type: geth
+    cl_type: lighthouse
+    validator_count: 32
+  - el_type: geth
+    cl_type: prysm
+    validator_count: 32
+
+additional_services:
+  - assertoor
+  - tx_spammer
+
+assertoor_params:
+  run_stability_check: false
+  run_block_proposal_check: false
+  run_blob_transaction_test: true
+  tests:
+    - 'https://raw.githubusercontent.com/lambdaclass/ethrex/refs/heads/assertoor-allaround-tx-check/test_data/assertoor/el-stability-check.yml'
```

Comment:
After review and before merging `assertoor-allaround-tx-check` will be replaced by `main`.

Reply:
Done